packages/libs/error-stack/rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
 # Please also update the badges in `README.md`s (`error-stack` and `error-stack-macros`), and `src/lib.rs`
```

Comment:
This is not captured by `renovate`

Reply:
True, for this PR we need to change it manually, but it may be possible to use renovate to automatically change this value.

Reply:
manually fixed in [5552bbe](https://github.com/hashintel/hash/pull/1865/commits/5552bbe75b2d34d39e49aa6a6ff7c5331e9eca16)

Reply:
Sadly, it's not, the commit neither did include the version bump in the readme of `error-stack-macros` nor the version in `src/lib.rs`.

Reply:
mb fixed in [accad94](https://github.com/hashintel/hash/pull/1865/commits/accad949e8d20a70cbf3b6858a24ca08fd30a60b) and [9233323](https://github.com/hashintel/hash/pull/1865/commits/923332399715a2b0ad6403a6ca02eae0ad0b0c0c)